src/llama_stack/distributions/ci-tests/build.yaml

```diff
@@ -56,4 +56,5 @@ image_type: venv
 additional_pip_packages:
 - aiosqlite
 - asyncpg
+- psycopg2-binary
```

Comment:
this is a bit unideal but we will leave it at that. we really need to decouple `ci-tests` distro from `starter`. on my plate.

Reply:
you mean removing `ci-tests` and using `starter` for testing? :p